guides/dev/features/balloontoolbar/README.md

```diff
@@ -6,18 +6,18 @@ For licensing, see LICENSE.md.
 # Balloon Toolbar
 
 <p class="requirements">
-	This feature was introduced in <strong>CKEditor 4.8</strong>. It is provided through the <a href="https://ckeditor.com/cke4/addon/balloontoolbar">Balloon Toolbar</a> plugin that is not included in the CKEditor preset available from the <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site and <a href="#!/guide/dev_plugins">need to be added to your custom build</a> with <a href="https://ckeditor.com/cke4/builder">CKBuilder</a>.
+	This feature was introduced in <strong>CKEditor 4.8</strong>. It is provided through the <a href="https://ckeditor.com/cke4/addon/balloontoolbar">Balloon Toolbar</a> plugin that is not included in any CKEditor preset available from the <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site and <a href="#!/guide/dev_plugins">needs to be added to your custom build</a> with <a href="https://ckeditor.com/cke4/builder">CKEditor 4 Online Builder</a>.
```

Comment:
@wojtekidd as for CKBuilder -> CKEditor 4 Online Builder - I'd say we should stick to "CKBuilder".

That would keep it in line with:

* https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_captionedimage
* https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_codesnippet
* https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_copyformatting

And few others. Does it make sense?

Reply:
It does make sense here - we just stopped calling it CKBuilder on the website and in its context. So yes please leave CKBuilder here 👌